client/src/app/components/identity/IdentitySearchInput.scss

```diff
@@ -97,4 +64,13 @@
       color: $warning !important;
     }
   }
+
+  svg {
+    position: absolute;
+    left: 16px;
+
+    path {
+      fill: var(--header-icon-color);
```

Comment:
@evavirseda  this color is not the same as the one used in the search icon on desktop (which is our reference component) 🙏🏼